src/Connection.ts

```diff
@@ -1784,15 +1784,15 @@ export class Connection<
 
 	/**
 	 * Delete a file of an adapter.
-	 * @param adapter The adapter name.
-	 * @param fileName The file name.
+	 * @param {string} namespace (this may be the adapter name, the instance name or the name of a storage object within the adapter).
+	 * @param fileName {string} The file name.
```

Comment:
```suggestion
	 * @param namespace (this may be the adapter name, the instance name or the name of a storage object within the adapter).
	 * @param fileName The file name.
```

Reply:
etc. etc. for the others too